spec/models/cart_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 describe Cart, type: :model do
   before(:each) do
+    mock_app_config
```

Comment:
We shouldn't need to mock this explicitly, I don't think, here or in the `spec/models/equipment_model_spec.rb`. I suspect that switching to our class method for retrieving the setting (see [above](https://github.com/YaleSTC/reservations/pull/1580/files#r66904083)) might allow things to work without an explicit mock.